src/schemaGenerator.ts

```diff
@@ -372,6 +373,17 @@ function addResolveFunctionsToSchema(
         return;
       }
 
+      if (type instanceof GraphQLEnumType) {
```

Comment:
Wait I'm really confused how this works with resolvers like:

```js
Color: {
  RED: '#EA3232'
},
```

Since:

```js
{
  RED: '#EA3232'
} instanceof GraphQLEnumType
```

Is definitely false, right?

Reply:
Yeah, I'd agree - but I don't think that's what's being checked for here ( this code is definitely being ran ) when I tried printing out `type`  here it says `"Color"` which I assume is the `description` on that class ( I'm still a little new to JS )